Source/Core/Core/HW/EXI/EXI_DeviceEthernetTAP.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <thread>
+
+#ifdef _WIN32
+#include <Windows.h>
+#include <vector>
+#endif
+
+#include "Common/Flag.h"
+#include "Core/HW/EXI/EXI_DeviceEthernetBase.h"
+
+namespace ExpansionInterface
+{
+class CEXIEthernetTAP : public CEXIEthernetBase
+{
+public:
+  ~CEXIEthernetTAP() override;
+
+protected:
+  bool Activate() override;
+  bool IsActivated() const override;
+  bool SendFrame(const u8* frame, u32 size) override;
+  bool RecvInit() override;
+  void RecvStart() override;
+  void RecvStop() override;
+
+private:
+  static void ReadThreadHandler(CEXIEthernetTAP* self);
+
+#ifdef _WIN32
+  HANDLE mHAdapter = INVALID_HANDLE_VALUE;
```

Comment:
You renamed the `Common::Flag` members down there to match our coding guidelines, but not the ones over here. Any particular reason behind that?